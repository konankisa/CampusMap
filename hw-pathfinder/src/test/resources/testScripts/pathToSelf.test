# Using Dijkstra's algorithm to check the
# path from a node to itself

# Creates a graph
CreateGraph graph

# Adds a node to the graph
AddNode graph node1

# Finds a path from the node to itself
FindPath graph node1 node1