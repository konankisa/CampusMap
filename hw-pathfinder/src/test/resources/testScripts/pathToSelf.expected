# Using Dijkstra's algorithm to check the
# path from a node to itself

# Creates a graph
created graph graph

# Adds a node to the graph
added node node1 to graph

# Finds a path from the node to itself
path from node1 to node1:
total cost: 0.000